src/Http/MiddlewareQueue.php

```diff
@@ -97,7 +97,7 @@ protected function resolve($middleware): MiddlewareInterface
     /**
      * Append a middleware to the end of the queue.
      *
-     * @param callable|string|array $middleware The middleware(s) to append.
+     * @param callable|string|array|\Psr\Http\Server\MiddlewareInterface $middleware The middleware(s) to append.
```

Comment:
This is one type too many. Should be remove the ability to pass array of middlewares? I don't see it overly useful. At least I didn't even remember that `add()` and other methods allow passing array.